src/components/SkillsContainer.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { SkillBar } from "./SkillBar";
+import { sampleData } from "../mock-data";
+
+export const SkillsContainer = () => {
+  const skills = sampleData.data.attributes.character.skills;
```

Comment:
I imagine this will be refactored into using the ApolloContext right? I'm still fuzzy on that aspect of the data distribution.

Reply:
can't wait!